packages/nuxi/src/commands/dev-child.ts

```diff
@@ -58,6 +59,15 @@ export default defineCommand({
       process.exit()
     })
 
+    const hostname = devContext.hostname
+    if (hostname) {
+      ctx.data ||= {}
+      ctx.data.overrides = defu(ctx.data.overrides, {
+        devServer: { cors: { origin: [devContext.proxy?.https ? 'https://' : `http://${hostname}`] } },
```

Comment:
Is not missing `${hostname}` at the 'https://'?
```suggestion
        devServer: { cors: { origin: [devContext.proxy?.https ? 'https://${hostname}' : `http://${hostname}`] } },
```

or
```suggestion
        devServer: { cors: { origin: [`http${devContext.proxy?.https ? 's' : ''}://'${hostname}`] } },
```

Reply:
nice spot - thank you! ❤️